data/net.sourceforge.wavbreaker.appdata.xml

```diff
@@ -3,7 +3,6 @@
 <component type="desktop-application">
   <id>net.sourceforge.wavbreaker</id>
   <metadata_license>FSFAP</metadata_license>
-  <content_rating>none</content_rating>
```

Comment:
According to [OARS](https://hughsie.github.io/oars/generate.html), I got:
```
  <content_rating type="oars-1.1" />
```

Should we put this in instead?

Reply:
Sure, if you have an actual value to supply, that’s even better!

Reply:
That makes sense!